processor/src/tasks/addToSearch.ts

```diff
@@ -105,11 +106,22 @@ function getReadmeAsText(readmeHTML) {
   return $.text()
 }
 
+async function getRepoDescription(ownerName: string, repoName: string) {
+  let description = ''
+  try {
+     const repo = await giteaDB.getRepoInfo(ownerName, repoName)
+     description = repo.description
+  } catch(e) {
+    log.warn("meilisearch: couldn't get repo description, falling back to empty string")
+  }
+  return description
+}
+
 /*
  * Subscribe to deletions on the repository table in the GiteaDB and delete
  * search index documents accordingly.
  */
-export function continuallySyncDeletions(giteaDB) {
+export function continuallySyncDeletions() {
```

Comment:
There is a reason for passing in the giteaDB as an argument everywhere: to allow us to write future tests without a DB connection.